data/reporter.htm

```diff
@@ -22,7 +22,7 @@
     </div>
 
     <div id="buttons">
-        <span id="appE10sEnabled"></span>
+        <span class="app-e10s-enabled"></span>
```

Comment:
You asked me to make this camel case when I added this :) (I'm ok with it, just saying)